Emby.Server.Implementations/Library/UserManager.cs

```diff
@@ -381,7 +377,7 @@ private static string GetPasswordResetProviderId(IPasswordResetProvider provider
 
         private IAuthenticationProvider GetAuthenticationProvider(User user)
         {
-            return GetAuthenticationProviders(user).First();
+            return GetAuthenticationProviders(user)[0];
```

Comment:
What's the advantage to this over .First()? The auth provider should always be accessible by index but with an IEnumerable that's not really for sure is it?

Reply:
It’s faster :p